otherlibs/win32unix/envir.c

```diff
@@ -21,23 +21,26 @@
 #include <caml/osdeps.h>
 
 #include <Windows.h>
+#include <Stdlib.h>
```

Comment:
stdlib (lowercase) is more standard (also for windows.h, btw) in our codebase.

Reply:
FWIW, an quick check in the museum shows that Microsoft use Windows.h and stdlib.h (or, of course, STDLIB.H if you're very, very old...)

However, the MinGW headers use windows.h so it's anyone's guess really... let's hope no one is crazy enough to try running the kernel in case sensitive mode :smile: